webpack/components/extensions/HostDetails/Tabs/ModuleStreamsTab/ModuleStreamsTab.js

```diff
@@ -0,0 +1,200 @@
+    stream: PropTypes.string.isRequired,
+    streamStatus: PropTypes.string.isRequired,
+  };
+
+  return (
+    <div>
+      <div id="modulestreams-tab">
+        <TableWrapper
+          {...{
+          metadata,
+          emptyContentTitle,
+          emptyContentBody,
+          emptySearchTitle,
+          emptySearchBody,
+          searchQuery,
+          updateSearchQuery,
+          fetchItems,
+          status,
+          rowActions,
+        }}
+          additionalListeners={[hostId]}
+          fetchItems={fetchItems}
+          autocompleteEndpoint={`/hosts/${hostId}/module_streams/auto_complete_search`}
+          foremanApiAutoComplete
+          rowsCount={results?.length}
+          variant={TableVariant.compact}
+        >
+          <Thead>
+            <Tr>
+              {columnHeaders.map(col =>
+                <Th key={col}>{col}</Th>)}
+            </Tr>
+          </Thead>
+          <Tbody>
+            {results?.map((moduleStream, index) => {
+              const {
+                id,
+                status: moduleStreamStatus,
+                name,
+                stream,
+                installed_profiles: installedProfiles,
+                install_status: installStatus,
+                module_spec: moduleSpec,
+              } = moduleStream;
+
+              let color;
+              let labelVariant;
+              let disabledText = false;
+              /* eslint-disable max-len */
+              let streamStatus = moduleStreamStatus.charAt(0).toUpperCase() + moduleStreamStatus.slice(1);
+              streamStatus = streamStatus.replace('Unknown', 'Default');
+              if (streamStatus === 'Default') {
+                color = 'gray';
+                labelVariant = 'outline';
+              } if (streamStatus === 'Disabled') {
+                color = 'gray';
+                labelVariant = 'filled';
+              } if (streamStatus === 'Enabled') {
+                color = 'green';
+                labelVariant = 'filled';
+              }
+              let installedProfile = installedProfiles;
+              if (installedProfiles.length === 0) {
+                installedProfile = 'No profile installed';
+              }
+              if (installedProfiles.length > 0) {
+                installedProfile = installedProfiles.map(x => x[0].toUpperCase() + x.substr(1));
+                installedProfile = installedProfile.join(', ');
+              }
+              if (streamStatus === 'Disabled' || streamStatus === 'Default') {
+                disabledText = true;
+              }
+
+            return (
+              <Tr key={`${id} ${index}`}>
+                <Td><a href={`/module_streams?search=module_spec%3D${moduleSpec}+and+host%3D${hostName}`}>{name}</a></Td>
+                <Td><Label color={color} variant={labelVariant}>{streamStatus}</Label></Td>
+                <Td>{stream}</Td>
+                <Td><EnabledIcon stream={streamStatus} streamStatus={installStatus} /></Td>
```

Comment:
this variable assignment looks a bit confusing because of the names.. maybe correlate the naming a bit better?